tools/chef/roles/db-mysql55-client.json

```diff
@@ -3,6 +3,18 @@
   "chef_type": "role",
   "json_class": "Chef::Role",
   "description": "MySQL 5.5 database client",
+  "default_attributes": {
+    "mysql-libs": {
+      "replace_package_targets": [
+        "mysql55-libs",
+        "Percona-Server-shared-51",
+        "mysqlclient16"
+      ]
+    },
+    "mysql-client": {
+      "replace_package_target": "mysql55"
+    }
+  },
```

Comment:
will the lack of a mysql-server replace target cause an existing mysql server to be removed if db-mysql*-server role not included?